src/main/sensors/initialisation.c

```diff
@@ -557,8 +557,8 @@ bool sensorsAutodetect(sensorAlignmentConfig_t *sensorAlignmentConfig, uint16_t
 {
     int16_t deg, min;
 
-    memset(&acc, sizeof(acc), 0);
-    memset(&gyro, sizeof(gyro), 0);
+    memset(&acc, 0, sizeof(acc));
+    memset(&gyro, 0, sizeof(gyro));
 
```

Comment:
thanks, but gcc 5.0 warning. This is also already fixed in tilt branch, but maybe it deserve an issue all for himself